### PR TITLE
Mads A. Hoofnagle - Adding My Bio

### PR DESCRIPTION
**Pull request by oneforae**
Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

I found this assignment relatively easy once I found and consulted the GitHub guide to using Markdown in RStudio. I decided to try my hand at coding everything that was included in the assignment to see if I could do it - it's rather frustrating that there (appears to be?) no place in RStudio to check if you've written the code correctly before you see it in GitHub. I made a mistake initially as I didn't place a space after the pound signs used to denote headings, so I had to go back and make another version with that corrected. It looks better to me now that I've fixed that issue.

### CODE REVIEW

**Review by jaycgibbs — COMMENTED** [review]
Nice use of all of the formatting steps (especially links, the video at the end was pretty cool). Also appreciated the alt text on the image you picked to embed. As for the preview bit in your note, you can preview the rich version of your text in RStudio by entering Control + Shift + K (or at least that works for me). Headers are well done. I don't really have any constructive feedback outside of the preview thing and maybe picking a superior MOBA haha

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Biography
+
+My name is Mads A. Hoofnagle. I am 24 years old and I live in Chicago, Illinois. I am in the Masters of Arts Program in Social Sciences at the University of Chicago. I have lived in *four different countries*: The U.S., Ireland, Iceland, and Denmark. I speak English and French fluently and some Icelandic, as well. 
+
+## Education
```
Comment (djanvier) [inline README.md:7]: Good use of headers here

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+## Academic Interests
+
+I used to be focused primarily on archaeology, but have since moved away from that in my current education to study medical anthropology. I am writing my current M.A. thesis on cultural facets of depression in Iceland. 
+
+## Hobbies and Extracurriculars
+
+I don't have much time right now to pursue my hobbies and interests outside of school, but I play a lot of video games. Currently, I've been spending a lot of time on: 
+
+1. Darkest Dungeon
```
Comment (djanvier) [inline README.md:19]: Nice list!

File: README.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+I also play in a Dungeons and Dragons campaign with some of my friends who live all around the world. We are currently hunting vampires! 
+
+When it comes to staying in shape, I like to go for a run or do some yoga. 
+
+Here is a picture of me playing DOTA 2 and having a beer:
+
+![Support Main](https://media.discordapp.net/attachments/400011446068576256/699409356294324254/IMG_2519.jpg?width=508&height=677)
+
+And [here](https://www.youtube.com/watch?v=ox_-BOCLv8s) is a link to an animation that was drawn to accompany a clip from one of my favorite podcasts, My Brother My Brother and Me. 
+
```
Comment (djanvier) [inline README.md:33]: The placement of these lines makes it a little hard to see. I would suggest placing this somewhere before the image or making the image smaller

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Reproducibility**         |     x      |              |            |
* Good use of the different Markdown syntaxes!
* Be mindful of any stylistic elements like spacing and readability
* You can view a rendered version of the README.md in R Studio by clicking on the **Preview** button at the top of the window. Though the rendered version in GitHub will look slightly different, R Studio will generate a version in HTML that is _very_ helpful and still useful for this purpose

**Pull request closed without merging**